src/components/interface/Tag/Tag.js

```diff
@@ -53,7 +53,7 @@ const Tag = ({
       <Icon
         name={icon}
         size={iconSize[size]}
-        iconPosition={children && `${iconPosition}`}
+        iconPosition={children ? iconPosition : 'center'}
```

Comment:
So by default `iconPosition` is not 'center' any more?

Reply:
the opposite, by default iconPosition was left, here it's without a children you need to center it
there is an example in the examples :)